executor/runtime/docker/capabilities.go

```diff
@@ -42,25 +44,21 @@ func setupAdditionalCapabilities(c *runtimeTypes.Container, hostCfg *container.H
 		}
 	}
 	seccompProfile := "default.json"
-	apparmorProfile := "docker-titus"
+	apparmorProfile := "docker_titus"
```

Comment:
Why the move to underscores for the profiles?

Reply:
https://gitlab.com/jo-so/apparmor/commit/0e0663e99e993e578e6092b5907fcbe5ae8988b1

Reply:
Dashes break apparmor 🤦‍♂️